src/gw2al_api.rs

```diff
@@ -4,54 +4,49 @@
 
 use std::os::raw::c_void;
 
-pub const GW2AL_CORE_FUNN_HASH_NAME: gw2al_hashed_name = 1;
-pub const GW2AL_CORE_FUNN_REG_FUN: gw2al_hashed_name = 2;
-pub const GW2AL_CORE_FUNN_UNREG_FUN: gw2al_hashed_name = 3;
-pub const GW2AL_CORE_FUNN_QUERY_FUN: gw2al_hashed_name = 4;
-pub const GW2AL_CORE_FUNN_FILL_VTBL: gw2al_hashed_name = 5;
-pub const GW2AL_CORE_FUNN_UNLOAD_ADDON: gw2al_hashed_name = 6;
-pub const GW2AL_CORE_FUNN_LOAD_ADDON: gw2al_hashed_name = 7;
-pub const GW2AL_CORE_FUNN_QUERY_ADDON: gw2al_hashed_name = 8;
-pub const GW2AL_CORE_FUNN_WATCH_EVENT: gw2al_hashed_name = 9;
-pub const GW2AL_CORE_FUNN_UNWATCH_EVENT: gw2al_hashed_name = 10;
-pub const GW2AL_CORE_FUNN_QUERY_EVENT: gw2al_hashed_name = 11;
-pub const GW2AL_CORE_FUNN_TRIGGER_EVENT: gw2al_hashed_name = 12;
-pub const GW2AL_CORE_FUNN_CLIENT_UNLOAD: gw2al_hashed_name = 13;
-pub const GW2AL_CORE_FUNN_LOG_TEXT: gw2al_hashed_name = 14;
-pub const GW2AL_CORE_FUNN_D3DCREATE_HOOK: gw2al_hashed_name = 15;
```

Comment:
These are not used anywhere and are wrong, so I removed them. lmk if you need these

Reply:
I literally copy pasted that from somewhere. dw about it